bigbluebutton-html5/imports/api/users/server/store/bannedUsers.js

```diff
@@ -0,0 +1,35 @@
+import Logger from '/imports/startup/server/logger';
+
+class BannedUsers {
+  constructor() {
+    Logger.debug('BannedUsers :: Initializing');
+    this.store = {};
+  }
+
+  init(meetingId) {
+    Logger.debug('BannedUsers :: init', meetingId);
+
+    if (!this.store[meetingId]) this.store[meetingId] = new Set();
+  }
+
+  add(meetingId, externalId) {
+    Logger.debug('BannedUsers :: add', { meetingId, externalId });
+    if (!this.store[meetingId]) this.store[meetingId] = new Set();
+
+    this.store[meetingId].add(externalId);
+  }
+
+  delete(meetingId) {
+    Logger.debug('BannedUsers :: delete', meetingId);
+    delete this.store[meetingId];
+  }
+
+  has(meetingId, externalId) {
+    Logger.debug('BannedUsers :: has', { meetingId, externalId });
+    if (!this.store[meetingId]) this.store[meetingId] = new Set();
```

Comment:
Not sure that it's a good idea to init again when checking for `has`. You already .init() on addMeeting

Reply:
It's a safe check to avoid exceptions, because we keep this store in memory and every time meteor restart we lose this structure.